docs/using-template-repos-for-assignments.md

```diff
@@ -0,0 +1,48 @@
+Importing starter code for assignments is a very useful feature and with GitHub recently shipping the template repositories, we can leverage them in Classroom to make importing starter code much easier and faster!
+
+### How to use template repositories for an assignment?
+
+#### Before you begin
```

Comment:
I feel like this section is kind-of a given (e.g. set up classroom) and unnecessary. @d12 what do you think?